cmd/skiff/top.go

```diff
@@ -66,14 +89,52 @@ func (h *FileHeap) Pop() interface{} {
 	return item
 }
 
+// getLayersByDiffID returns layer blob infos filtered by user-provided diffIDs
+// by looking up diffIDs and mapping to manifest layers
+func getLayersByDiffID(manifestLayers []types.BlobInfo, allDiffIDs []digest.Digest, filterDiffIDs []string) ([]types.BlobInfo, []digest.Digest, error) {
```

Comment:
Isn't `BlobInfo.Digest` usable in this context?

Reply:
What do you mean?

Reply:
the `manifestLayers` array is of type `types.BlobInfo`, which has a field `Digest`. Why not use that `Digest` instead of passing the additional `allDiffIDs` array with the digests as a separate parameter?